iib/common/tracing.py

```diff
@@ -7,8 +7,8 @@
 
 Usage:
     @instrument_tracing()
-    def func():
-        pass
+      def func():
+          pass
```

Comment:
This was the only way I found to write this comment, where Sphinx was not throwing errors.

Reply:
Could you please share the error you were getting without this extra tab?

Reply:
`/iib/common/tracing.py:docstring of iib.common.tracing:9:Unexpected indentation.`